packages/clerk-js/src/ui/components/SignUp/SignUpStart.tsx

```diff
@@ -192,6 +193,10 @@ function _SignUpStart(): JSX.Element {
 
     card.setLoading();
     card.setError(undefined);
+
+    const redirectUrl = buildSSOCallbackURL(ctx, displayConfig.signUpUrl);
+    const redirectUrlComplete = ctx.afterSignInUrl || displayConfig.afterSignUpUrl;
```

Comment:
@yourtallness This is using both `afterSignInUrl` and `afterSignUpUrl`. Is this intentional?

Reply:
Fixed in #1148, thanks!